src/sql/workbench/browser/modal/media/modal.css

```diff
@@ -176,20 +291,20 @@
 
 .vs .modal.flyout-dialog .dialog-message.error,
 .vs-dark .modal.flyout-dialog .dialog-message.error {
-	background-color:#B62E00 !important;
```

Comment:
not a huge deal, but I've been always using upper case for color hex, any reason you are changing them to lower case?

Reply:
No particular reason. Easy enough to use uppercase here.